pipeline/tier_executor.groovy

```diff
@@ -278,6 +278,18 @@ node(nodeName) {
             writeYaml file: "${env.WORKSPACE}/${dirName}/metadata.yaml", data: content
             sharedLib.uploadResults(dirName, "${env.WORKSPACE}/${dirName}")
 
+            currentBuild.result = "SUCCESS"
```

Comment:
Can we avoid `currentBuild.result` attribute here. Instead, can we use `buildStatus`.

Reply:
new variable is introduced